asapdiscovery-alchemy/asapdiscovery/alchemy/utils.py

```diff
@@ -158,18 +158,22 @@ def action_network(
         return actioned_tasks
 
     def network_status(
-        self, planned_network: FreeEnergyCalculationNetwork
+        self,
+        planned_network: Optional[FreeEnergyCalculationNetwork] = False,
```

Comment:
`None` is probably more idiomatic 

```suggestion
        planned_network: Optional[FreeEnergyCalculationNetwork] = None,
```